packages/components/src/DatePicker/DatePicker.stories.tsx

```diff
@@ -81,8 +82,10 @@ export const Controlled: Story = {
           <DatePicker
             label="Date Picker"
             value={value}
-            onChange={setValue}
+            defaultValue={value}
```

Comment:
Do I need defaultValue here? It is a controlled state so I don't need it or?

Reply:
we need it because to set the default... otherwise its not shown :(